src/styles/body.scss

```diff
@@ -0,0 +1,6 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the `box-sizing` property should be set globally using the universal selector `*`, not on the `body` element. Move this property to a global level to ensure that all elements inherit `box-sizing: border-box;` by default.